assets/app/view/game/dividend.rb

```diff
@@ -180,18 +184,25 @@ def dividend_chart
 
         table_props = {
           style: {
-            margin: '0.5rem 0 0 0',
+            margin: '0.5rem 0 0.5rem 0',
             textAlign: 'left',
             border: '1px solid black',
             borderCollapse: 'collapse',
           },
         }
 
+        header_props = {
+          style: {
+            backgroundColor: color_for(:bg2),
+            border: '1px solid black',
```

Comment:
should this be a user preference collor for the dark mode users?

Reply:
Dunno. Is there an example I can look at?

Reply:
For what it's worth, it's no different than how the shading is done at the top and bottoms of the corporate cards.

Reply:
Isn't that what color_for() does?

Reply:
Got it. Removed hard-coded 'black'